worker.go

```diff
@@ -264,6 +265,27 @@ func (w *worker) getAndDeleteUniqueJob(job *Job) *Job {
 	return jobWithArgs
 }
 
+func (w *worker) alive(job *Job) bool {
+	conn := w.pool.Get()
+	defer conn.Close()
+
+	key := redisKeyKilledJob(w.namespace, job.ID)
+
+	killed, err := redis.Int(conn.Do("GET", key))
+	if err != nil && err != redis.ErrNil {
+		logError("worker.alive.get", err)
+		return true
+	}
+
+	_, err = conn.Do("DEL", key)
+	if err != nil {
+		logError("worker.alive.del", err)
+		return true
```

Comment:
Should this return `killed != 1` though?  I would think that the `alive` state should depend on if we can delete the key or not.